### PR TITLE
Remaining secure message tests converted to standalone

### PR DESCRIPTION
**Pull request by AdeHarrison**
# Motivation and Context
Remaining secure message tests converted to standalone

# How to test?
make setup acceptance_tests
and then
make secure_messaging_acceptance_tests

# Links
[Trello](https://trello.com/c/T0p0Gilm/386-t386convert-remaining-secure-messaging-features-to-standalone)

### CODE REVIEW

**Comment by AdamHawtin** [comment]
[T386:Convert remaining secure messaging features to standalone](https://trello.com/c/T0p0Gilm/386-t386convert-remaining-secure-messaging-features-to-standalone)

File: acceptance_tests/features/pages/create_message_internal.py [hunk 1]
```diff
@@ -24,11 +23,11 @@ def get_ru_details_attributes():
...
 
 
-def found_respondent_details():
-    return {'ru_ref': RESPONDENT_DETAILS.get_ru_ref(),
+def found_respondent_details(context):
+    return {'ru_ref': context.short_name,
             'to': 'first_name last_name',
-            'to_uuid': RESPONDENT_DETAILS.get_respondent_id(),
-            'to_ru_id': RESPONDENT_DETAILS.get_ru_id()}
+            'to_uuid': context.respondent_user_name,
```
Comment (AdamHawtin) [inline acceptance_tests/features/pages/create_message_internal.py]: This value should be the respondent's party uuid, not email. I don't think it actually gets used anywhere though, so perhaps we should remove it completely here?

Comment (AdeHarrison) [reply acceptance_tests/features/pages/create_message_internal.py]: deleted "to_ru_id" & "to_uuid" as not used anywhere

File: acceptance_tests/features/steps/create_message_internal.py [hunk 2]
```diff
@@ -2,25 +2,28 @@
...
-    assert create_message_internal.RESPONDENT_DETAILS.get_ru_ref() is not None
+def user_has_found_respondent(context):
+    context.party_id = find_case_by_enrolment_code(context.iac)['partyId']
+
+    assert context.respondent_user_name is not None
+    assert context.party_id is not None
+    assert context.short_name is not None
 
 
 @when("they choose to send them a secure message and navigated to the 'send message' page")
```
Comment (AndrewTorrance) [inline acceptance_tests/features/steps/create_message_internal.py]: use of quotation marks inside given, then and when clauses should be restricted to passing variables otherr wise it is confusing e.g should be 
either 
@when("they choose to send them a secure message and navigated to the send message page")
when no params passed

or 
When("they choose to send them a secure message and navigated to the 'send message' page")
@when(@when("they choose to send them a secure message and navigated to the {page_name} page")
def navigate_to_send_message(_, page_name):     # Passes value "send message" in the page_name variable
.....

When using the quotations to send a variable into the function/clause

File: acceptance_tests/features/steps/create_message_internal.py [hunk 3]
```diff
@@ -2,25 +2,28 @@
...
 @given("the user is on the send message page")
-def navigate_to_send_message(_):
-    create_message_internal.go_to()
+def navigate_to_send_message(context):
+    create_message_internal.go_to(context)
 
 
 @then("the \'To\' field is populated with the respondent's name")
```
Comment (AndrewTorrance) [inline acceptance_tests/features/steps/create_message_internal.py]: See above regarding quotes in clause strings

File: acceptance_tests/features/steps/create_message_internal.py (hunk 2 above)
Comment (AdeHarrison) [reply acceptance_tests/features/steps/create_message_internal.py]: fixed

File: acceptance_tests/features/steps/create_message_internal.py (hunk 3 above)
Comment (AdeHarrison) [reply acceptance_tests/features/steps/create_message_internal.py]: fixed

**Pull request merged**